### PR TITLE
Update passing-values-to-generators.md

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/ecmascript-2015/generators-proxy/passing-values-to-generators.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/ecmascript-2015/generators-proxy/passing-values-to-generators.md?ref=f1fe0a3ee0d963671c0fde3c2936ad0ab7779556)

File: javascript/ecmascript-2015/generators-proxy/passing-values-to-generators.md [hunk 1]
```diff
@@ -24,58 +18,58 @@ links:
...
-g.next().value //2
-g.next().value //3
-g.next(0).value //1
+g.next().value; // 1
+g.next().value; // 2
+g.next().value; // 3
+g.next(0).value; // 1
 ```
 
 There is however also a gotcha to watch out for when passing values to generators.
```
Comment (Stefan-Stojanovic) [inline javascript/ecmascript-2015/generators-proxy/passing-values-to-generators.md]:
```suggestion
There is also a gotcha to watch out for when passing values to generators.
```

Not sure if "however" is needed here

File: javascript/ecmascript-2015/generators-proxy/passing-values-to-generators.md [hunk 2]
```diff
@@ -24,58 +18,58 @@ links:
 ---
 ## Content
 
-Generators next method also accepts a parameter.
+Generator's `next` method also accepts a parameter.
 
-When you pass in a parameter it is assigned to the result of the **yield** statement (if you don’t pass anything in this is undefined).
+When you pass in a parameter it is assigned to the result of the `yield` statement (if you don’t pass anything in it is `undefined`).
 
 We could for example use this functionality to reset our number sequence when 0 is passed in:
```
Comment (Stefan-Stojanovic) [inline javascript/ecmascript-2015/generators-proxy/passing-values-to-generators.md]:
```suggestion
We could, for example, use this functionality to reset our number sequence when 0 is passed in:
```

File: javascript/ecmascript-2015/generators-proxy/passing-values-to-generators.md [hunk 3]
```diff
@@ -13,8 +9,6 @@ aspects:
   - introduction
   - workout
 
-inAlgoPool: false
-
 links:
   - '[MDN - Generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator){website}'
```
Comment (kapnobatai137) [inline javascript/ecmascript-2015/generators-proxy/passing-values-to-generators.md]: does this sound better?
```suggestion
  - '[Generators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Generator){documentation}'
```

File: javascript/ecmascript-2015/generators-proxy/passing-values-to-generators.md (hunk 1 above)
Comment (kapnobatai137) [reply javascript/ecmascript-2015/generators-proxy/passing-values-to-generators.md]: how about this?
```suggestion
There is, however, also a gotcha to watch out for when passing values to generators.
```

Comment (nem035) [reply javascript/ecmascript-2015/generators-proxy/passing-values-to-generators.md]: I think we should use either "however" or "also", but not both

**Pull request merged**